k256/src/arithmetic/projective.rs

```diff
@@ -108,6 +144,7 @@ impl ProjectivePoint {
         let yz_pairs = ((self.y + &self.z) * &(other.y + &other.z)) + &n_yy_zz;
         let xz_pairs = ((self.x + &self.z) * &(other.x + &other.z)) + &n_xx_zz;
 
+        // TODO can remove these to simplify patch (from bigint1)
```

Comment:
Can remove this comment to simplify applying just bigint1 patch and this new one (maybe less conflict)

Reply:
```suggestion
```

Reply:
Made the change, but maybe an open question of what we do with that logic now given it's obsolete with bigint2

Reply:
In my opinion we should just drop it. It's unreachable code, it's not useful with the current architecture, and if we discover we _do_ actually need it later for some reason, git will remember the past for us.

Reply:
Fair enough, was assuming it might make applying just this commit more difficult in the future, but probably not a case where it matters